pipelines/qc/submit.sh

```diff
@@ -1,8 +1,6 @@
 #! /bin/bash
-
-mkdir -p _log
-snakemake -T $@ \
-    --cluster 'qsub -v PATH="/home/njh/miniconda3/bin:$PATH" -j y -o $(pwd)/_log/ -b n {params.pe} -l {params.req},h=!foxtrot.well.ox.ac.uk -S /bin/bash' \
+snakemake --snakefile `dirname $0`/Snakefile $@ \
+    --cluster 'qsub -v PATH="/home/njh/miniconda3/bin:$PATH" -j y -b n -l {params.req},h=!foxtrot.well.ox.ac.uk -S /bin/bash' \
```

Comment:
`PATH="/home/njh/miniconda3/bin:$PATH"` seems to be quite specific. Can we use binder environment instead?

Reply:
oops well spotted!